bundles/AdminBundle/src/EventListener/AdminAuthenticationDoubleCheckListener.php

```diff
@@ -1,167 +0,0 @@
-<?php
-declare(strict_types=1);
-
-/**
- * Pimcore
- *
- * This source file is available under two different licenses:
- * - GNU General Public License version 3 (GPLv3)
- * - Pimcore Commercial License (PCL)
- * Full copyright and license information is available in
- * LICENSE.md which is distributed with this source code.
- *
- *  @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
- *  @license    http://www.pimcore.org/license     GPLv3 and PCL
- */
-
-namespace Pimcore\Bundle\AdminBundle\EventListener;
-
-use Pimcore\Bundle\AdminBundle\Controller\DoubleAuthenticationControllerInterface;
-use Pimcore\Bundle\AdminBundle\EventListener\Traits\ControllerTypeTrait;
-use Pimcore\Bundle\AdminBundle\Security\User\TokenStorageUserResolver;
-use Pimcore\Bundle\CoreBundle\EventListener\Traits\PimcoreContextAwareTrait;
-use Pimcore\Http\Request\Resolver\PimcoreContextResolver;
-use Pimcore\Http\RequestMatcherFactory;
-use Pimcore\Tool\Authentication;
-use Symfony\Component\EventDispatcher\EventSubscriberInterface;
-use Symfony\Component\HttpFoundation\Request;
-use Symfony\Component\HttpFoundation\RequestMatcherInterface;
-use Symfony\Component\HttpKernel\Event\ControllerEvent;
-use Symfony\Component\HttpKernel\Exception\AccessDeniedHttpException;
-use Symfony\Component\HttpKernel\KernelEvents;
-
-/**
- * Handles double authentication check for pimcore controllers after the firewall did to make sure the admin interface is
- * not accessible on configuration errors. Unauthenticated routes are not double-checked (e.g. login).
- *
- * @internal
- */
```

Comment:
Should we really kick that, it ensures the admin is not accessible even if someone messed up the security config? 🤔

Reply:
if someone messes up the security config, it is basically their fault.

Reply:
... and still, they will blame Pimcore. I also vote for keeping that.

Reply:
they will, but it is still their fault. we cannot protect from stupidity. I absolutely don't see any reason that is needed when symfony handles that!